README.md

```diff
@@ -11,138 +11,62 @@ LDAP (Lightweight Directory Access Protocol) is a vendor-neutral software protoc
 
-### APIs associated with LDAP
+## Quickstart
 
-- **add**: Creates an entry in a directory server.
-- **modify**: Updates information of an entry in a directory server.
-- **modifyDN**: Renames an entry in a directory server.
-- **compare**: Determines whether a given entry has a specified attribute value.
-- **search**: Returns a record containing search result entries and references that match the given search parameters.
-- **searchWithType**: Returns a list of entries that match the given search parameters.
-- **delete**: Removes an entry from a directory server.
-- **close**: Unbinds from the server and closes the LDAP connection.
+To use the LDAP connector in your Ballerina project, modify the `.bal` file as follows.
 
-#### `add` API
+### Step 1: Import the module
 
-Creates an entry in a directory server.
+Import the `ballerina/ldap` module into your Ballerina project.
 
 ```ballerina
 import ballerina/ldap;
-
-public function main() returns error? {
-    anydata user = {
-        "objectClass": "user",
-        "sn": "New User",
-        "cn": "New User",
-        "givenName": "New User",
-        "displayName": "New User",
-        "userPrincipalName": "newuser@ad.windows",
-        "userAccountControl": "544"
-    };
-    ldap:LdapResponse val = check ldapClient->add(userDN, user);
-}
-```
-
-#### `modify` API
-
-Updates information of an entry.
-
-```ballerina
-import ballerina/ldap;
-
-public function main() returns error? {
-    anydata user = {
-        "sn": "User",
-        "givenName": "Updated User",
-        "displayName": "Updated User"
-    };
-    _ = check ldapClient->modify(userDN, user);
-}
-```
-
-#### `modifyDN` API
-
-Renames an entry in a directory server.
-
-```ballerina
-import ballerina/ldap;
-
-public function main() returns error? {
-    ldap:LdapResponse modifyDN = check ldapClient->modifyDN(userDN, "CN=Test User2", true);
-}
-```
-
-#### `compare` API
-
-Determines whether a given entry has a specified attribute value.
-
-```ballerina
-import ballerina/ldap;
-
-public function main() returns error? {
-    ldap:LdapResponse compare = check ldapClient->compare(userDN, "givenName", "Test User1");
-}
-```
-
-#### `search` API
-
-Returns a record containing search result entries and references that match the given search parameters.
-
-```ballerina
-import ballerina/ldap;
-
-public function main() returns error? {
-    ldap:SearchResult value = check ldapClient->search("DC=ad,DC=windows", "(givenName=Test User1)", SUB);
-}
 ```
 
-#### `searchWithType` API
-
-Returns a list of entries that match the given search parameters.
+### Step 2: Instantiate a new connector
 
 ```ballerina
-import ballerina/ldap;
-
-public function main() returns error? {
-    anydata[] value = check ldapClient->searchWithType("DC=ad,DC=com", "(givenName=Test User1)", ldap:SUB);
-}
+ldap:Client ldapClient = check new ({
```

Comment:
Shall we define the configurables for `hostName`, `port`, `domainName`, and `password` ?